package.json

```diff
@@ -73,10 +76,10 @@
     "prettier": "1.16.4",
     "prompt": "^1.0.0",
     "radium": "^0.22.0",
-    "react": "16.4.2",
-    "react-dom": "16.4.2",
-    "react-styleguidist": "9.0.9",
-    "react-test-renderer": "16.4.2",
+    "react": "^16.12.0",
```

Comment:
we should prob make sure this is a minor version change, as react updating is not insignificant (bundle sizes and all)

Reply:
good idea 👍 , I think this would actually somehow be the first non-patch release.